### PR TITLE
[proposal] suggest render repo description by markdown language

### PR DESCRIPTION
**Pull request by a1012112796**
mark it as a draft because I'm not sure it's necessay, it's just a proposal, waiting more responses, thanks.

### CODE REVIEW

File: models/repo.go [hunk 1]
```diff
@@ -990,13 +990,8 @@ func (repo *Repository) getUsersWithAccessMode(e Engine, mode AccessMode) (_ []*
...
 // DescriptionHTML does special handles to description and return HTML string.
-func (repo *Repository) DescriptionHTML() template.HTML {
-	desc, err := markup.RenderDescriptionHTML([]byte(repo.Description), repo.HTMLURL(), repo.ComposeMetas())
-	if err != nil {
-		log.Error("Failed to render description for %s (ID: %d): %v", repo.Name, repo.ID, err)
-		return template.HTML(markup.Sanitize(repo.Description))
-	}
-	return template.HTML(markup.Sanitize(string(desc)))
+func (repo *Repository) DescriptionHTML() string {
+	return string(markdown.Render([]byte(repo.Description), repo.HTMLURL(), repo.ComposeMetas()))
```
Comment (lunny) [inline models/repo.go:994]: I think `markup.RenderDescriptionHTML` has done that.

Comment (a1012112796) [reply models/repo.go:994]: No, it's a little different with markdown.
example:
image


 I wander why design this special ruler to render it.

**Comment by silverwind** [comment]
I think I'm fine with allowing single-line markdown-rendered content there but I'm not sure if we want to allow multi-line stuff. as that could destroy our layout.

**Comment by mrsdizzie** [comment]
PR that implemented the current version seems suspiciously specific yet no details in the PR itself -- @zeripath does this need to be done a particular way for security or other concern?

**Comment by lunny** [comment]
I think since only one line allowed currently, the requirement for the description to render as markdown is little.

**Pull request closed without merging**